test/gtest/group_conv3d_wrw.hpp

```diff
@@ -30,6 +30,7 @@
 std::vector<Conv3DTestCase> ConvTestConfigs()
 { // g   n   c   d    h   w   k   z  y  x pad_x pad_y pad_z stri_x stri_y stri_z dia_x dia_y dia_z
     return {{1, 128, 64, 14, 28, 28, 64, 3, 3, 3, 1, 1, 1, 1, 1, 1, 1, 1, 1, miopenConvolution},
+            {2, 128, 32, 28, 28, 28, 32, 3, 3, 3, 0, 0, 0, 2, 2, 2, 1, 1, 1, miopenConvolution},
```

Comment:
This row causes a very long execution time or a hang. Not sure. If I reduce the `N` and `C, K` values, I see a numerical failure just like `bwd` pass.